package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "passport-auth0",
-  "version": "0.6.1",
```

Comment:
Can you rebase this commit on top of the latest master? I've updated the lock file so that may be separate from this PR.